internal/learn/future/assembly.md

```diff
@@ -0,0 +1,35 @@
+---
+id: assembly
+title: Assembly
+description: Introduction to the emerging permissionless Smart Contracts network
+keywords:
+  - smart contracts
+  - permissionless
+  - ASMB
+---
+
+![Assembly](/img/learn/future/assembly.png 'Click to see the full-size image.')
+
+In December 2021, IOTA Foundation [introduced](https://blog.assembly.sc/announcing-assembly-and-the-asmb-token/) Assembly, a permissionless Smart Contracts network. Assembly will function as an intermediate layer for permissionless and interoperable Smart Contracts. It will also have its own token called [ASMB](https://assembly.sc/token). The Token will help secure the network.
+This introduction should only give you a brief overview of Assembly. For more information, check the official [Website](https://assembly.sc) or the [Assembly Wiki](https://wiki.assembly.com).
+
+## What is Assembly?
```

Comment:
Maybe just cut everything from there and on? Looks like doing the same job twice (here and in Assembly Wiki).

Reply:
Not sure about that. I can drop `What is Assembly`. But it was really important to me to explain the value of IOTA in Assembly as many seem to not understand that and imo it's the most important part to understand the future of IOTA. 🤔

Reply:
I think this article will be better if you focus on why Assembly and Tangle benefit on each other, yes. "Assembly needs Tangle to communicate and secure chains, Tangle benefits from Assembly because chains need mana to commit state and the best way to always have Mana is to run a tangle node".

Reply:
So you would ony add this single paragraph instead of everything under ``What is Assembly?`?
I'm fine with that but then we need this conntent in the assembly wiki. If it isn't already and I just didn't find it yet ;)
Wdyt?

Reply:
Not necessary all in a single paragraph, but I think you can cut explaining what Assembly is (beyond saying that's a new network), and focus on why Assembly empowers IOTA and not overshadows it, since this is the real question you want to answer.

Reply:
What do you think about the changes now 😅 ?